### DISCUSSION TITLE
Problems about negative samples

### DISCUSSION THREAD

**Discussion by Ashinhuang** [facebookresearch/detectron2#3053] [Q&A]
Dear,
## Instructions To Reproduce the 🐛 Bug:
1. Full runnable code or full changes you made:
![image](https://user-images.githubusercontent.com/33593302/118491688-d218b280-b751-11eb-9791-5b7bb8be0562.png)

I need to train the negative sample, the negative sample format is shown above, category_id:0, segmentation:[]
 but  it is wrong ,
So what should I write in negative sample format?

**Comment by ppwwyyxx** [comment]
Write nothing. Provide no annotations for images that have no annotations. https://detectron2.readthedocs.io/en/latest/tutorials/datasets.html has mentioned this.

**Reply by shining-love** [reply]
hello have you solved this problem?

**Reply by ExtReMLapin** [reply]
@shining-love just do what he says, just provide images with no annotations and do `cfg.DATALOADER.FILTER_EMPTY_ANNOTATIONS = False`
